### DISCUSSION TITLE
Why redis from your repo?

### DISCUSSION THREAD

**Discussion by samof76** [OT-CONTAINER-KIT/redis-operator#429] [General]
I see that in your sample deployments you seem to use.

```
quay.io/opstree/redis:v7.0.5
```

Any reason why you use your redis?

**Comment by shubham-cmyk** [comment]
The image we provide is lightweight for a comparison 

1. Redis Official Image  - 117MB
2. Opstree redis image - 31.1MB

Also I am not sure whether the official image provides a health check or not if it does you can use that one. 

@samof76

**Reply by samof76** [reply]
Ok can you link me to the Dockerfile from where this docker image is built?

**Reply by shubham-cmyk** [reply]
Sure : https://github.com/OT-CONTAINER-KIT/redis/blob/master/Dockerfile
